<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNewMemberForm</class>
 <widget class="QWidget" name="addNewMemberForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>423</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>322</width>
    <height>423</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>322</width>
    <height>423</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add new member</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(14, 109, 205);</string>
  </property>
  <widget class="QLineEdit" name="lineEditCode">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>Code</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLastName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPhoneNumber">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>18</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 4);
background-color: rgb(157, 234, 255);
font: 14pt &quot;Blacksword&quot;;</string>
   </property>
   <property name="text">
    <string>NEW</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>18</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 4);
background-color: rgb(157, 234, 255);
font: 14pt &quot;Blacksword&quot;;</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(160, 255, 248);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Qt/digitalMarketManagment/join_membership_ad_member_become-512.png</normaloff>../../Qt/digitalMarketManagment/join_membership_ad_member_become-512.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
